module/stats.py

```diff
@@ -0,0 +1,22 @@
+"""/stats command"""
+from telegram import Update
+from telegram.ext import CallbackContext
+from module.shared import SELECT, STATS_MESSAGE
+from module.create_connection import connect_and_execute
+
+def stats(update: Update, context: CallbackContext) -> None:
+    """called by /stats
+        It returns the number of books that are now on sale
+
+        Args:
+            update: update event
+            context: context passed by the handler"""
+
+    query: str = "SELECT COUNT(*) FROM Market"
+    chat_id: int = update.effective_chat.id
+    params = ()
+    num: int = connect_and_execute(
+        context=context, chat_id=chat_id, query=query, params=params, operation=SELECT
+    )[0][0]
```

Comment:
it's no safe the [0][0] at the end, and if the result is null or empty?

Reply:
In the case that the value is null or empty can I write for example "Non c'è nessun libro in questo momento nel market"?

Reply:
I think so